chartbeat.php

```diff
@@ -244,6 +244,51 @@ function chartbeat_is_validjson($json_str) {
 	}
 }
 
+function chartbeat_configs() {
+	$cb_configs['domain'] = esc_js( apply_filters( 'chartbeat_config_domain', chartbeat_get_display_url (get_option('home')) ));
```

Comment:
Should we auto-strip "www." from the start of domains here since our AMP pinger can't do that automatically?